packages/protocol/scripts/truffle/deployedGrants.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
.gitignore?

Reply:
Need it for the cli tests

Reply:
Why not allow users to optionally specify no deployed grants file?